CitiesUIService/build.gradle

```diff
@@ -41,4 +46,31 @@ dependencyManagement {
     imports {
         mavenBom "org.springframework.cloud:spring-cloud-dependencies:Brixton.SR5"
     }
+}
+
+ext {
```

Comment:
I'm not sure that correctly understood a comment. I made a definition of rest endpoint in the file swagger.properties. commit f601692